src/main/java/frc/robot/subsystems/NAVX.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.subsystems;
+
+
+
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+import com.revrobotics.ColorSensorV3;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class NAVX extends SubsystemBase {
+ 
+    private static double blue;
+    private static double red;
+    private static double green;
+    private static ColorSensorV3 colorSensor;
+    public final static int threshold = 300;
+    private static DigitalInput distanceSensor;
+    private static DigitalInput distanceSensorFar;
+    private static boolean shortDistance;
+    private static boolean longDistance;
+
+    private static int thresholdDistance = 800;
+
+
+
+    private static String color;
+
```

Comment:
added every sensor but the NavX lmao

Reply:
See how to add NavX as well as common methods here: https://www.kauailabs.com/public_files/navx-mxp/apidocs/java/com/kauailabs/navx/frc/AHRS.html